README.md

```diff
@@ -83,6 +83,43 @@ non-zero length value.  Logs are sent to syslog and standard error by default.
 | SYSLOG_ERROR_PRIORITY | syslog.error     | `logger -p <value>`           |
 | VARDIR                | /var/lib/startup | Durable alternative to TMPDIR |
 
+# Automated Tests
+
+Automated tests are run inside of a docker image.  The initial tests are modeled
+as specification tests validating expected behavior.  [bats][bats] is the test
+framework run inside the container.
+
+## Spec Tests
+
+### Build the bats container image
+
+Build the image used to validate specification tests.  This command builds an
+image with bats installed, suitable for running the tests.
+
+```sh
+make docker_build_bats
+```
+
+### Validate Behavior Specifications
+
+Validate the expected behavior using the image containing the bats tools.  This
+command runs the container and executes bats against each of the test suite
+files in `test/specs/`
```

Comment:
Tiny nitpick - the path is `test/spec`, not `test/specs`.

Reply:
Great eye, thanks!

Reply:
Fixed in a3cea47...7e56437 HEAD -> test_structure (forced update)